collective/catalogcleanup/browser.py

```diff
@@ -18,8 +18,21 @@
 logger = logging.getLogger('collective.catalogcleanup')
 
 
+def safe_path(item):
+    try:
+        return item.getPath()
+    except KeyError:
```

Comment:
you could add `AttributeError` also to make it safer as sometimes a brain can be `None`.

Reply:
I'm not sure that can happen here. Well, further on in this file we do catch `AttributeError` because that can happen for references. Alright, there is no harm in catching an extra error. I will do so and then merge.